src/containers/status/Status.js

```diff
@@ -105,6 +106,34 @@ function Status ({ className }) {
     </Tooltip>
   );
 
+  const renderWatcherSynced = (
+    <Tooltip
+      title={
+        isWatcherSynced
+          ? 'Watcher is synced with the child chain'
```

Comment:
I think it would help to have a better explanation of what syncing actually means

Reply:
great Idea, and even better: we should have it documented on the dev portal